add_pypi_package.sh

```diff
@@ -105,13 +94,13 @@ add_pypi_to_comps() {
 }
 
 add_pypi_to_manifest() {
-	if [[ $TITO_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
+	if [[ $KOJI_TAG == "foreman-nightly-nonscl-rhel7" ]] ; then
 		local section="foreman_nonscl_packages"
-	elif [[ $TITO_TAG == "foreman-plugins-nightly-nonscl-rhel7" ]] ; then
+	elif [[ $KOJI_TAG == "foreman-plugins-nightly-nonscl-rhel7" ]] ; then
 		local section="plugin_nonscl_packages"
-	elif [[ $TITO_TAG == "katello-nightly-rhel7" ]] ; then
+	elif [[ $KOJI_TAG == "katello-nightly-rhel7" ]] ; then
 		local section="katello_packages"
-	elif [[ $TITO_TAG == "katello-pulpcore-nightly-el7" ]] ; then
+	elif [[ $KOJI_TAG == "katello-pulpcore-nightly-el7" ]] ; then
 		local section="pulpcore_packages"
```

Comment:
This should be updated to EL8. Alternatively, we can not talk about a koji tag but require the user to specify the section in the manifest.

Reply:
I thought about this as well and figured it as a future update to persue on top of this change.